### PR TITLE
created develop branch, changed file permissions to execute

### PR DESCRIPTION
**Pull request by nankspn**
created develop branch, changed file permissions to execute, added a new wrapper script to create and launch rocket

### CODE REVIEW

**Comment by kspnec** [comment]
Verified create-and-launch-rocket script, ok to merge now.
But modularity is needed to launch more rockets.
work them as a new feature.

File: nasa-rocket/create-and-launch-rocket [hunk 1]
```diff
@@ -0,0 +1,8 @@
+mkdir lunar-mission
```
Comment (kspnec) [inline nasa-rocket/create-and-launch-rocket:1]: add improved functionality to pass new rocket mission as CLI arguments.

Comment (nankspn) [reply nasa-rocket/create-and-launch-rocket:1]: ok, will fix in next feature improvement.

Comment (kspnec) [reply nasa-rocket/create-and-launch-rocket:1]: fine, will fix in next release. Good to merge now.

**Pull request merged**